### PR TITLE
EP: Seguimiento de Paciente

### PR DESCRIPTION
**Pull request by plammel**
### Requerimiento

Seguimiento de casos epidemiología

https://proyectos.andes.gob.ar/browse/EP-85
https://proyectos.andes.gob.ar/browse/EP-86
https://proyectos.andes.gob.ar/browse/EP-65
### Funcionalidad desarrollada 

1. 
2. 
3. 

### UserStories llegó a completarse

- [ ] Si
- [ ] No

### Requiere actualizaciones en la base de datos

- [x] Si POST http://localhost:3002/api/modules/georeferencia/areasProgama/
- [ ] No

### CODE REVIEW

**Comment by liquid36** [comment]
USUARIO: plammel  
BUILD NUMBER: [4366](http://condorpiedra.andes.gob.ar:85/4366/mochawesome.html)
CYPRESS RUN: [3389](https://dashboard.cypress.io/projects/xr7gft/runs/3389/specs)
TEST START: 2021-06-17T16:07:52.063Z
TOTAL: 396
SUCCESS: 381
FAIL: 14
SKIPPED: 1

File: modules/georeferencia/routes/georeferencia.ts [hunk 1]
```diff
@@ -34,4 +35,14 @@ router.get('/autocompletar/', async (req, res, next) => {
     }
 });
 
+
+router.post('/areasProgama', async (req, res, next) => {
```
Comment (sychus) [inline modules/georeferencia/routes/georeferencia.ts:39]: Tenemos que proteger esta ruta!

**Review by sychus — COMMENTED** [review]
Voy a proteger esa ruta.... y pusheo. El resto lo vi bien!

**Comment by liquid36** [comment]
USUARIO: plammel  
BUILD NUMBER: [4373](http://condorpiedra.andes.gob.ar:85/4373/mochawesome.html)
CYPRESS RUN: [3395](https://dashboard.cypress.io/projects/xr7gft/runs/3395/specs)
TEST START: 2021-06-17T21:43:45.812Z
TOTAL: 396
SUCCESS: 386
FAIL: 9
SKIPPED: 1

File: modules/georeferencia/areaPrograma.schema.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import * as mongoose from 'mongoose';
+
+export const AreaProgramaSchema = new mongoose.Schema(
+    {
+        nombre:  mongoose.SchemaTypes.Mixed,
```
Comment (martinebucarey) [inline modules/georeferencia/areaPrograma.schema.ts:5]: por que no es un string el nombre??

**Comment by liquid36** [comment]
USUARIO: plammel  
BUILD NUMBER: [4385](http://condorpiedra.andes.gob.ar:85/4385/mochawesome.html)
CYPRESS RUN: [3404](https://dashboard.cypress.io/projects/xr7gft/runs/3404/specs)
TEST START: 2021-06-18T17:01:26.277Z
TOTAL: 399
SUCCESS: 391
FAIL: 7
SKIPPED: 1

File: modules/georeferencia/areaPrograma.schema.ts (hunk 2 above)
Comment (plammel) [reply modules/georeferencia/areaPrograma.schema.ts]: Corregido!

**Comment by liquid36** [comment]
USUARIO: plammel  
BUILD NUMBER: [4394](http://condorpiedra.andes.gob.ar:85/4394/mochawesome.html)
CYPRESS RUN: [3413](https://dashboard.cypress.io/projects/xr7gft/runs/3413/specs)
TEST START: 2021-06-22T18:04:35.178Z
TOTAL: 399
SUCCESS: 391
FAIL: 7
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: plammel  
BUILD NUMBER: [4397](http://condorpiedra.andes.gob.ar:85/4397/mochawesome.html)
CYPRESS RUN: [3416](https://dashboard.cypress.io/projects/xr7gft/runs/3416/specs)
TEST START: 2021-06-22T20:30:48.140Z
TOTAL: 399
SUCCESS: 396
FAIL: 2
SKIPPED: 1

**Review by martinebucarey — APPROVED** [review]

**Comment by liquid36** [comment]
USUARIO: plammel  
BUILD NUMBER: [4399](http://condorpiedra.andes.gob.ar:85/4399/mochawesome.html)
CYPRESS RUN: [3418](https://dashboard.cypress.io/projects/xr7gft/runs/3418/specs)
TEST START: 2021-06-23T13:47:48.595Z
TOTAL: 399
SUCCESS: 396
FAIL: 2
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: plammel  
BUILD NUMBER: [4402](http://condorpiedra.andes.gob.ar:85/4402/mochawesome.html)
CYPRESS RUN: [3421](https://dashboard.cypress.io/projects/xr7gft/runs/3421/specs)
TEST START: 2021-06-23T16:02:34.144Z
TOTAL: 399
SUCCESS: 398
FAIL: 0
SKIPPED: 1

File: modules/forms/forms-epidemiologia/forms-epidemiologia-schema.ts [hunk 3]
```diff
@@ -16,12 +17,52 @@ export const FormsEpidemiologiaSchema = new mongoose.Schema({
...
+                currentDate.getMonth() === birthDate.getMonth() && currentDate.getDate() < birthDate.getDate()) {
+                years--;
+            }
+            edad = years;
+        }
+        return edad;
+    };
+
+    const seccionClasificacion = ficha.secciones.find(s => s.name === 'Tipo de confirmación y Clasificación Final');
+    let clasificacionfinal = seccionClasificacion.fields.find(f => f.clasificacionfinal)?.clasificacionfinal;
```
Comment (martinebucarey) [inline modules/forms/forms-epidemiologia/forms-epidemiologia-schema.ts:53]: Por el nuevo requerimiento de la ficha parcial probablemente haya que tener un control mas
   `  let clasificacionfinal = seccionClasificacion?.fields.find(f => f.clasificacionfinal)?.clasificacionfinal;`

**Review by sychus — APPROVED** [review]

**Comment by liquid36** [comment]
USUARIO: plammel  
BUILD NUMBER: [4472](http://condorpiedra.andes.gob.ar:85/4472/mochawesome.html)
CYPRESS RUN: [3492](https://dashboard.cypress.io/projects/xr7gft/runs/3492/specs)
TEST START: 2021-07-01T17:12:08.432Z
TOTAL: 390
SUCCESS: 388
FAIL: 1
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: plammel  
BUILD NUMBER: [4475](http://condorpiedra.andes.gob.ar:85/4475/mochawesome.html)
CYPRESS RUN: [3495](https://dashboard.cypress.io/projects/xr7gft/runs/3495/specs)
TEST START: 2021-07-01T17:49:32.316Z
TOTAL: 398
SUCCESS: 397
FAIL: 0
SKIPPED: 1

**Pull request merged**